packages/frontend/src/styles/index.scss

```diff
@@ -1,4 +1,4 @@
 @import './bootstrap-config';
 @import '../../node_modules/bootstrap/scss/bootstrap';
 
-$test: .1;
+$test: 0.1;
```

Comment:
This variable was only added to test the linting for the pre-commit hook and can be removed entirely.